packages/components/src/Persona.purs

```diff
@@ -87,6 +89,18 @@ getUserEntitlements :: UserAuth -> Aff (Array String)
 getUserEntitlements auth =
   callApi usersApi "usersUuidEntitlementGet" [ unsafeToForeign auth.userId ] $ authHeaders auth.userId auth
 
+getPaywallOpenings :: UserAuth -> Aff (Array PaywallOpening)
+getPaywallOpenings auth =
+  callApi entitlementsApi "entitlementsAllowGet" [] $ authHeaders UUID.emptyUUID auth
+
+openPaywall :: UserAuth -> AllowEntitlementsQuery -> Aff Unit
```

Comment:
The style in this file has been to always have `UserAuth` as the last parameter, this could be changed to match with that.